schemas/webfilters.json

```diff
@@ -83,6 +83,23 @@
                 "items": {
                     "type": "integer"
                 }
+            },
+            "cert_name": {
+                "type": "string",
+                "title": "Certificate Username",
+                "description": "Certificate Username",
```

Comment:
Add a bit more description. Maybe
"Provide certificate username for Webfiler VM. Defaults to webfilter" and then regenerate the files